stream_alert_cli/test.py

```diff
@@ -752,13 +752,13 @@ def run_tests(options, context):
         log_mem_hanlder = get_log_memory_hanlder()
 
         # Check if the rule processor should be run for these tests
-        test_rules = (set(run_options.get('processor')).issubset({'rule', 'all'})
+        test_rules = (not run_options.get('processor').isdisjoint({'rule', 'all'})
```

Comment:
Can you provide more content about why do you choose to use `isdisjoint` over `issubset`?

Reply:
@chunyong-lin Do you want a code comment or just some context in the PR body?

Reply:
Code comment is good for me 😄  Thanks!